### PR TITLE
CI: add EVE

### PR DESCRIPTION
**Pull request by carlito-scality**
**Component**:UI

**Description**:
- Add EVE to execute snapshot test in the CI

**Design**:

**Breaking Changes**:

- [] Breaking Changes

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: eve/workers/pod-unit-tests/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN yum install -y epel-release \
+    && yum install -y gcc \
+    make \
+    python-devel \
+    python-pip \
+    python36 \
+    python36-devel \
+    python36-pip \
+    hg \
```
Comment (jbertran) [inline eve/workers/pod-unit-tests/Dockerfile]: Are you sure we need `hg`?

File: eve/workers/pod-unit-tests/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    git \
+    nodejs \
```
Comment (jbertran) [inline eve/workers/pod-unit-tests/Dockerfile]: May be interesting to use a `node:${NODE_VERSION}` image instead, now that this works.

File: eve/workers/pod-unit-tests/pod.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "scality-core-ui-unit-tests-worker"
+spec:
+  containers:
+  - name: worker
```
Comment (jbertran) [inline eve/workers/pod-unit-tests/pod.yaml]: use the `env` key for the `CI` environment variable here, so you don't need to change `package.json`.

File: yarn.lock [hunk 4]
```diff
@@ -1143,20 +1143,20 @@
   dependencies:
     regenerator-runtime "^0.13.2"
```
Comment (jbertran) [inline yarn.lock]: Not sure where these lock modifications come from, could you commit them in a separate commit?

**Review by jbertran — COMMENTED** [review]
Excellent job!

Your idea of storybook updates on gh-pages in post-merge merits a ticket too :)

**Review by jbertran — APPROVED** [review]
partyparrot.gif

**Comment by carlito-scality** [comment]
@bert-e approve

**Pull request merged**